### ISSUE TITLE
Archive repo?

### ISSUE THREAD

**Issue by durack1** [PCMDI/climlib#12]
@pochedls just doing a cleanup of PCMDI repos, should this be archived (or deleted)?

**Comment by pochedls** [comment]
I think it is time to say farewell to climlib. I'm happy to have this deleted – it never really grew how we envisioned. I think if we try an effort like this again, we should start from scratch (and use xcdat).

**Comment by durack1** [comment]
Agreed, sad but true. Are there any reusable bits in here that warrants archival, or have you already migrated these somewhere else, using xcdat underneath?

**Comment by pochedls** [comment]
I don't think so – a lot of the code never really reached maturity – I would write it from scratch...
